rlgl.go

```diff
@@ -29,10 +34,18 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
-	if CheckEvents(coreclient) && CheckNodeEvents(coreclient, t) {
-		os.Exit(0)
-	} else {
-		os.Exit(1)
+
+	if !*f {
+		if CheckEvents(coreclient) && CheckNodeEvents(coreclient, t) {
+			os.Exit(0)
+		} else {
+			os.Exit(1)
+		}
+	}
+	for *f {
+		CheckEvents(coreclient)
+		CheckNodeEvents(coreclient, t)
+		time.Sleep(time.Duration(*sleep) * time.Second)
```

Comment:
wonder if the last state (when the ctrl+c is pressed) should be our exit code?

Reply:
or @chentex maybe we watch until an error occurs (if it occurs)... Thoughts?

Reply:
That could be a good option, catching the last status before ctrl + c might be tricky, but for sure we can exit(1) once we catch an error.

Reply:
Updated with the changes